configcatclient/evaluationdetails.py

```diff
@@ -7,17 +7,35 @@ def __init__(self,
                  user=None,
                  is_default_value=False,
                  error=None,
-                 matched_evaluation_rule=None,
-                 matched_evaluation_percentage_rule=None):
+                 matched_targeting_rule=None,
+                 matched_percentage_option=None):
+        # Key of the feature flag or setting.
         self.key = key
+
+        # Evaluated value of the feature flag or setting.
         self.value = value
+
+        # Variation ID of the feature flag or setting (if available).
         self.variation_id = variation_id
+
+        # Time of last successful config download.
         self.fetch_time = fetch_time
+
+        # The User Object used for the evaluation (if available).
         self.user = user
+
+        # Indicates whether the default value passed to the setting evaluation methods like ConfigCatClient.get_value,
+        # ConfigCatClient.get_value_details, etc. is used as the result of the evaluation.
         self.is_default_value = is_default_value
+
+        # Error message in case evaluation failed.
         self.error = error
-        self.matched_evaluation_rule = matched_evaluation_rule
-        self.matched_evaluation_percentage_rule = matched_evaluation_percentage_rule
+
+        # The targeting rule which was used to select the evaluated value (if any).
+        self.matched_targeting_rule = matched_targeting_rule
+
+        # The percentage option which was used to select the evaluated value (if any).
```

Comment:
UX team made changes to this text: https://github.com/configcat/.net-sdk/pull/84/commits/262f33bc4e90ccb376e9595feb897805682ebb9c#diff-e53563e3fe29251a2c13adec55b8021190b6355694e18fc29d3c9164301ed85c